translations/ja.po

```diff
@@ -122,58 +128,64 @@ msgstr ""
 msgid "Could not find art for `${ name }`!"
 msgstr ""
 
-#: src\commands\deck.ts:132
+#: src\commands\deck.ts:198
 #, javascript-format
 msgid "${ counts.Monster } Monster"
 msgid_plural "${ counts.Monster } Monsters"
 msgstr[0] ""
+msgstr[1] ""
```

Comment:
Still affected. I think I'll add a sed or awk command to the translations documentation to remove these from ja, ko, zh